feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt

```diff
@@ -0,0 +1,130 @@
+    val serviceRequest = "1"
+    val amenityReservations = "1"
+    val eventList = listOf("Coffee + Donuts", "Pool party", "Discounts")
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(sizes.regularSize)
+            .verticalScroll(rememberScrollState())
+    ) {
+        Row(modifier = Modifier.fillMaxWidth()) {
+            Text(
+                modifier = Modifier
+                    .align(Alignment.CenterVertically)
+                    .weight(1f),
+                text = stringResource(R.string.welcome_back, username),
+                style = LPTypography.titleMedium,
+            )
+            LpBadgeButton(
+                badgeNumber = badgeNumberNotification,
+                showBadge = showBadgeNotification,
+                imageVector = Icons.Filled.Notifications
+            ) {
+                Log.e("tag", "go to notification")
+            }
+            LpBadgeButton(
+                badgeNumber = badgeNumberPerson,
+                showBadge = showBadgePerson,
+                imageVector = Icons.Filled.Person
+            ) {
+                Log.e("tag", "go to person")
+            }
+        }
+        Spacer(modifier = Modifier.size(sizes.minorRegularSize))
+        LpGenericCard(
+            modifier = Modifier
+                .fillMaxWidth(),
+            title = stringResource(R.string.current_balance),
+            details = stringResource(R.string.go_to_payments),
+            accountNumber = currentBalance,
+            currency = currentBalanceCurrency
+        ) {
+            Log.e("tag", "Go to payments")
+        }
+        Spacer(modifier = Modifier.size(sizes.regularSize))
+        LpGenericCard(
+            title = stringResource(R.string.service_request),
+            details = stringResource(R.string.view_detail),
+            accountNumber = serviceRequest,
+        ) {
+            Log.e("tag", "go to service request")
+        }
+        Spacer(modifier = Modifier.size(sizes.regularSize))
+        LpGenericCard(
+            title = stringResource(R.string.amenity_reservations),
+            details = stringResource(R.string.view_detail),
+            accountNumber = amenityReservations
+        ) {
+            Log.e("tag", "go to amenity reservations")
+        }
+        Spacer(modifier = Modifier.size(sizes.regularSize))
+        Text(
+            text = stringResource(R.string.happening_today),
+            style = LPTypography.titleMedium,
+            color = MaterialTheme.colorScheme.tertiary
+        )
+        LazyRow(
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            items(eventList) { event ->
+                LPGenericElevatedCard(
+                    title = event,
+                    description = "This is mock data",
+                    buttonText = "Go to Screen",
+                    color = getRandomColor(Random.nextInt(0, 3))
+                ) {
+                    Log.e("tag", event)
```

Comment:
Please add a //TODO comment to tell us which action is going to be executed in this button.